openslides/motions/serializers.py

```diff
@@ -337,6 +293,47 @@ def validate(self, data):
         return data
 
 
+class MotionCommentSectionSerializer(ModelSerializer):
+    """
+    Serializer for motion.models.MotionCommentSection objects.
+    """
+    read_groups = IdPrimaryKeyRelatedField(
+        many=True,
+        required=False,
+        queryset=Group.objects.all())
```

Comment:
Here you can use get_user_model().groups...

Reply:
the type of `get_user_model().groups.field.realted_model` is `django.contrib.auth.Group`. I don't know what's wrong there and I won't search deeper. I just created a `get_group_model` in out `auth.py` that works exactly like the django's one for users.